src/modules/verifiable-credentials/eip712.types.ts

```diff
@@ -0,0 +1,92 @@
+export const verifiableCredentialEIP712Types = {
+  EIP712Domain: [],
```

Comment:
Could you please elaborate what is the purpose of this type and why it is deleted domain types before signing?

Reply:
This type is required by Spruce library, but `ethers` don't want it.